app/js/components/labOrderEntry/LabDraftOrder.jsx

```diff
@@ -32,18 +32,36 @@ export class LabDraftOrder extends PureComponent {
         },
       );
       return (
-        <li className="draft-list small-font" key={shortid.generate()}>
-          <span>{orderName}</span>
-          <span className="stay-right">
-            <a className="action-btn" id="draft-toggle-btn" href="#" onClick={() => this.handleToggleDraftOrderUgency(order)}>
+        <li className="draft-item small-font" key={shortid.generate()}>
+          <span className="draft-name">{orderName}</span>
+          <span className="action-btn">
+            <a
+              id="draft-toggle-btn" href="#" onClick={(event) => {
+                event.preventDefault();
+                this.handleToggleDraftOrderUgency(order);
+              }
+              }
+            >
               <i className={iconClass} title="Urgency" />
             </a>
           </span>
+          <span className="action-btn right">
+            <a
+              id="draft-discard-btn" href="#" onClick={(event) => {
+                event.preventDefault();
+                const action = !isPanel ? 'single' : 'panel';
+                handleDraftDiscard(order, action);
+              }
+              }
```

Comment:
Hi @rhenshaw56 
I suggest you compose this: 
```
onClick={(event) => {
     event.preventDefault();
     const action = !isPanel ? 'single' : 'panel';
     handleDraftDiscard(order, action);
}
```
in a separate function and call the function when needed.

Reply:
@IAMOTZ  done!